--- origsrc/pygame-1.9.3/config_unix.py	2016-08-20 15:28:55.000000000 -0500
+++ src/pygame-1.9.3/config_unix.py	2017-03-14 14:20:45.224442000 -0500
@@ -159,27 +159,27 @@ def main():
             inc_porttime = os.environ.get('PORTMIDI_INC_PORTTIME')
             portmidi_as_porttime = True if inc_porttime in ['1', 'True'] else False
         else:
-            if os.path.exists('/etc/redhat-release'):
+            if os.path.exists('/etc/redhat-release') or sys.platform == 'cygwin':
                 portmidi_as_porttime = True
             else:
                 portmidi_as_porttime = False
 
         if portmidi_as_porttime:
-            return Dependency('PORTTIME', 'porttime.h', 'libportmidi.so', ['portmidi'])
+            return Dependency('PORTTIME', 'porttime.h', 'libportmidi', ['portmidi'])
         else:
-            return Dependency('PORTTIME', 'porttime.h', 'libporttime.so', ['porttime'])
+            return Dependency('PORTTIME', 'porttime.h', 'libporttime', ['porttime'])
 
     porttime_dep = get_porttime_dep()
 
     DEPS = [
         DependencyProg('SDL', 'SDL_CONFIG', 'sdl-config', '1.2', ['sdl']),
-        Dependency('FONT', 'SDL_ttf.h', 'libSDL_ttf.so', ['SDL_ttf']),
-        Dependency('IMAGE', 'SDL_image.h', 'libSDL_image.so', ['SDL_image']),
-        Dependency('MIXER', 'SDL_mixer.h', 'libSDL_mixer.so', ['SDL_mixer']),
+        Dependency('FONT', 'SDL_ttf.h', 'libSDL_ttf', ['SDL_ttf']),
+        Dependency('IMAGE', 'SDL_image.h', 'libSDL_image', ['SDL_image']),
+        Dependency('MIXER', 'SDL_mixer.h', 'libSDL_mixer', ['SDL_mixer']),
         Dependency('PNG', 'png.h', 'libpng', ['png']),
         Dependency('JPEG', 'jpeglib.h', 'libjpeg', ['jpeg']),
         Dependency('SCRAP', '', 'libX11', ['X11']),
-        Dependency('PORTMIDI', 'portmidi.h', 'libportmidi.so', ['portmidi']),
+        Dependency('PORTMIDI', 'portmidi.h', 'libportmidi', ['portmidi']),
         porttime_dep,
         DependencyProg('FREETYPE', 'FREETYPE_CONFIG', 'freetype-config', '2.0',
                        ['freetype'], '--ftversion'),
