--- origsrc/pygame-1.9.1release/config_unix.py	2009-05-26 16:15:24.000000000 -0500
+++ src/pygame-1.9.1release/config_unix.py	2009-11-17 02:35:00.453452300 -0600
@@ -138,15 +138,15 @@ def main():
     print ('\nHunting dependencies...')
     DEPS = [
         DependencyProg('SDL', 'SDL_CONFIG', 'sdl-config', '1.2', ['sdl']),
-        Dependency('FONT', 'SDL_ttf.h', 'libSDL_ttf.so', ['SDL_ttf']),
-        Dependency('IMAGE', 'SDL_image.h', 'libSDL_image.so', ['SDL_image']),
-        Dependency('MIXER', 'SDL_mixer.h', 'libSDL_mixer.so', ['SDL_mixer']),
+        Dependency('FONT', 'SDL_ttf.h', 'libSDL_ttf', ['SDL_ttf']),
+        Dependency('IMAGE', 'SDL_image.h', 'libSDL_image', ['SDL_image']),
+        Dependency('MIXER', 'SDL_mixer.h', 'libSDL_mixer', ['SDL_mixer']),
         DependencyProg('SMPEG', 'SMPEG_CONFIG', 'smpeg-config', '0.4.3', ['smpeg']),
         Dependency('PNG', 'png.h', 'libpng', ['png']),
         Dependency('JPEG', 'jpeglib.h', 'libjpeg', ['jpeg']),
         Dependency('SCRAP', '', 'libX11', ['X11']),
-        Dependency('PORTMIDI', 'portmidi.h', 'libportmidi.so', ['portmidi']),
-        Dependency('PORTTIME', 'porttime.h', 'libportmidi.so', ['portmidi']),
+        Dependency('PORTMIDI', 'portmidi.h', 'libportmidi', ['portmidi']),
+        Dependency('PORTTIME', 'porttime.h', 'libportmidi', ['portmidi']),
         #Dependency('GFX', 'SDL_gfxPrimitives.h', 'libSDL_gfx.so', ['SDL_gfx']),
     ]
     if not DEPS[0].found:
--- origsrc/pygame-1.9.1release/src/display.c	2009-07-13 01:36:15.000000000 -0500
+++ src/pygame-1.9.1release/src/display.c	2009-11-17 02:35:21.544689100 -0600
@@ -338,7 +338,7 @@ get_wm_info (PyObject* self)
 /*scary #ifdef's match SDL_syswm.h*/
 #if (defined(unix) || defined(__unix__) ||              \
      defined(_AIX) || defined(__OpenBSD__)) &&          \
-    (defined(SDL_VIDEO_DRIVER_X11) && !defined(__CYGWIN32__) && \
+    (defined(SDL_VIDEO_DRIVER_X11) && \
      !defined(ENABLE_NANOX) && !defined(__QNXNTO__))
 
     tmp = PyInt_FromLong (info.info.x11.window);
--- origsrc/pygame-1.9.1release/src/event.c	2009-07-13 01:36:15.000000000 -0500
+++ src/pygame-1.9.1release/src/event.c	2009-11-17 02:35:50.779140100 -0600
@@ -346,7 +346,7 @@ dict_from_event (SDL_Event* event)
          */
 #if (defined(unix) || defined(__unix__) || defined(_AIX)        \
      || defined(__OpenBSD__)) &&                                \
-    (defined(SDL_VIDEO_DRIVER_X11) && !defined(__CYGWIN32__) &&         \
+    (defined(SDL_VIDEO_DRIVER_X11) &&         \
      !defined(ENABLE_NANOX) && !defined(__QNXNTO__))
 
         //printf("asdf :%d:", event->syswm.msg->event.xevent.type);
