--- origsrc/pygame-1.9.6/setup.py	2019-04-25 02:14:48.000000000 -0400
+++ src/pygame-1.9.6/setup.py	2019-05-12 23:46:23.383303000 -0400
@@ -274,15 +274,6 @@ for f in glob.glob(os.path.join('src_py'
     if not f[-3:] == '.py' and not f[-4:] == '.doc' and os.path.isfile(f):
         pygame_data_files.append(f)
 
-#tests/fixtures
-add_datafiles(data_files, 'pygame/tests',
-              ['test',
-                  [['fixtures',
-                      [['xbm_cursors',
-                          ['*.xbm']],
-                       ['fonts',
-                          ['*.ttf', '*.otf', '*.bdf', '*.png']]]]]])
-
 #examples
 add_datafiles(data_files, 'pygame/examples',
               ['examples',
@@ -610,25 +601,11 @@ PACKAGEDATA = {
        "packages":    ['pygame',
                        'pygame.threads',
                        # 'pygame._sdl2',
-                       'pygame.tests',
-                       'pygame.tests.test_utils',
-                       'pygame.tests.run_tests__tests',
-                       'pygame.tests.run_tests__tests.all_ok',
-                       'pygame.tests.run_tests__tests.failures1',
-                       'pygame.tests.run_tests__tests.incomplete',
-                       'pygame.tests.run_tests__tests.infinite_loop',
-                       'pygame.tests.run_tests__tests.print_stderr',
-                       'pygame.tests.run_tests__tests.print_stdout',
-                       'pygame.tests.run_tests__tests.incomplete_todo',
-                       'pygame.tests.run_tests__tests.exclude',
-                       'pygame.tests.run_tests__tests.timeout',
-                       'pygame.tests.run_tests__tests.everything',
                        'pygame.docs',
                        'pygame.examples'],
        "package_dir": {'pygame': 'src_py',
                        # 'pygame._sdl2': 'src_py/_sdl2',
                        'pygame.threads': 'src_py/threads',
-                       'pygame.tests': 'test',
                        'pygame.docs': 'docs',
                        'pygame.examples': 'examples'},
        "headers":     headers,
