--- origsrc/pygame-1.9.1release/src/_camera.c	2009-03-15 14:30:41.000000000 -0500
+++ src/pygame-1.9.1release/src/_camera.c	2009-11-17 02:51:12.010794800 -0600
@@ -35,7 +35,7 @@
  
 #include "camera.h"
 
-#if defined(__unix__)
+#if defined(__linux)
 #else
     #define V4L2_PIX_FMT_RGB24 1
     #define V4L2_PIX_FMT_RGB444 1
@@ -127,7 +127,7 @@ PyObject* surf_colorspace (PyObject* sel
 /* list_cameras() - lists cameras available on the computer */
 PyObject* list_cameras (PyObject* self, PyObject* arg)
 {
-#if defined(__unix__)
+#if defined(__linux)
     PyObject* ret_list;
     PyObject* string;
     char** devices;
@@ -158,7 +158,7 @@ PyObject* list_cameras (PyObject* self, 
 /* start() - opens, inits, and starts capturing on the camera */
 PyObject* camera_start (PyCameraObject* self)
 {
-#if defined(__unix__)
+#if defined(__linux)
     if (v4l2_open_device(self) == 0) {
         if (v4l_open_device(self) == 0) {
             v4l2_close_device(self);
@@ -192,7 +192,7 @@ PyObject* camera_start (PyCameraObject* 
 /* stop() - stops capturing, uninits, and closes the camera */
 PyObject* camera_stop (PyCameraObject* self)
 {
-#if defined(__unix__)
+#if defined(__linux)
     if (v4l2_stop_capturing(self) == 0)
         return NULL;
     if (v4l2_uninit_device(self) == 0)
@@ -208,7 +208,7 @@ PyObject* camera_stop (PyCameraObject* s
 PyObject* camera_get_controls (PyCameraObject* self)
 {
     int value;
-#if defined(__unix__)    
+#if defined(__linux)    
     if (v4l2_get_control(self->fd, V4L2_CID_HFLIP, &value))
         self->hflip = value;
     
@@ -234,7 +234,7 @@ PyObject* camera_set_controls (PyCameraO
     
     if (!PyArg_ParseTupleAndKeywords(arg, kwds, "|iii", kwids, &hflip, &vflip, &brightness))
         return NULL;
-#if defined(__unix__)        
+#if defined(__linux)        
     if (v4l2_set_control(self->fd, V4L2_CID_HFLIP, hflip))
         self->hflip = hflip;
         
@@ -256,7 +256,7 @@ PyObject* camera_get_size (PyCameraObjec
 /* query_image() - checks if a frame is ready */
 PyObject* camera_query_image (PyCameraObject* self)
 {
-#if defined(__unix__)
+#if defined(__linux)
     return PyBool_FromLong(v4l2_query_buffer(self));
 #endif
     Py_RETURN_NONE;
@@ -286,7 +286,7 @@ PyObject* camera_get_image (PyCameraObje
         return RAISE (PyExc_ValueError, 
                       "Destination surface not the correct width or height.");
     }
-#if defined(__unix__)
+#if defined(__linux)
     Py_BEGIN_ALLOW_THREADS;
     if (!v4l2_read_frame(self, surf))
         return NULL;
@@ -306,7 +306,7 @@ PyObject* camera_get_image (PyCameraObje
 /* get_raw() - returns an unmodified image as a string from the buffer */
 PyObject* camera_get_raw(PyCameraObject* self)
 {
-#if defined(__unix__)
+#if defined(__linux)
     return v4l2_read_raw(self);
 #endif
     Py_RETURN_NONE;
--- origsrc/pygame-1.9.1release/src/camera.h	2008-10-09 21:37:10.000000000 -0500
+++ src/pygame-1.9.1release/src/camera.h	2009-11-17 02:51:41.390835300 -0600
@@ -20,7 +20,7 @@
 #include "pygame.h"
 #include "pygamedocs.h"
 
-#if defined(__unix__)
+#if defined(__linux)
     #include <structmember.h>
     #include <stringobject.h>
     #include <stdio.h>
--- origsrc/pygame-1.9.1release/src/camera_v4l.c	2008-09-27 23:31:42.000000000 -0500
+++ src/pygame-1.9.1release/src/camera_v4l.c	2009-11-17 02:52:16.030883100 -0600
@@ -16,7 +16,7 @@
   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
   
 */
-#if defined(__unix__)
+#if defined(__linux)
 #include "camera.h"
 
 /*
--- origsrc/pygame-1.9.1release/src/camera_v4l2.c	2008-09-27 23:31:42.000000000 -0500
+++ src/pygame-1.9.1release/src/camera_v4l2.c	2009-11-17 02:52:27.500898900 -0600
@@ -17,7 +17,7 @@
   
 */
 
-#if defined(__unix__)
+#if defined(__linux)
 
 #include "camera.h"
 
